onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -3921,27 +3921,12 @@ def aten_hspmm(mat1: TensorType, mat2: TensorType) -> TensorType:
     raise NotImplementedError()
 
 
-@torch_op("aten::hstack")
```

Comment:
Maybe just delete the registration and keep the code? Put a TODO or NOTE saying that this is slow so we rely on PyTorch decomposition. Just in case we will need them, and this could save the efforts?

Reply:
ok will do

Reply:
Done